lib/common/ServiceInfo.ts

```diff
@@ -0,0 +1,46 @@
+import ServiceVersionModel from './models/ServiceVersionModel'; 
+
+/**
+ * Encapsulates the functionality to get the information about the service such as
+ * version info.
+ */
+export default class ServiceInfo {
+
+    private static readonly packageJson = require('../../package.json');
+
+    private serviceName: string;
+
+    constructor(serviceName: string) {
+        this.serviceName = serviceName;
+    }
+
+    /**
+     * Gets the service version from the package.json file.
+     */
+    public getServiceVersion(): ServiceVersionModel {
+        return {
+            name: this.serviceName,
+            version: ServiceInfo.packageJson.version
+        };        
+    }
+
+    /**
+     * Gets an 'empty' service version object.
+     */
+    public static getEmptyServiceVersion(): ServiceVersionModel {

```

Comment:
Design: consider not having this "dummy" method. I think `tryGetServiceVersion()` can itself return either:

`undefined` or
```
{
  name:  'service_name',
  version: 'undefined'
}

Reply:
Refactoring this code out to the new ServiceVersionFetcher.ts